pydantic/types.py

```diff
@@ -1013,6 +1013,15 @@ def _serialize(v: Any) -> str:
                 return v.__name__
             elif hasattr(v, '__module__') and hasattr(v, '__name__'):
                 return f'{v.__module__}.{v.__name__}'
+            # Handle special cases for sys.XXX streams
+            # if we see more of these, we should consider a more general solution
```

Comment:
I think in general serializing the imported object to a string is a hard (even impossible without special casing as it is done here) problem, so I'm fine with the proposed solution.

Reply:
Cool, thanks!